doc/nrf/known_issues.rst

```diff
@@ -659,6 +659,17 @@ Subsystems
 Bluetooth® LE
 =============
 
+.. rst-class:: v1-8-0
+DRGN-16506: Requesting an MPSL timeslot with MPSL_TIMESLOT_HFCLK_CFG_NO_GUARANTEE will take 43us more than in the previous versions.
+
+.. rst-class:: v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0
+
+DRGN-16506: REM does not stop the HF Clock in between events when 'no prepare' is configured
+  When requesting an MPSL timeslot with MPSL_TIMESLOT_HFCLK_CFG_NO_GUARANTEE, the HF clock will be kept running between timeslot events.
+  This results in increased power consumption when idle.
+
+  **Workaround:** Use MPSL_TIMESLOT_HFCLK_CFG_XTAL_GUARANTEED instead of MPSL_TIMESLOT_HFCLK_CFG_NO_GUARANTEE to requested a timeslot.
```

Comment:
```suggestion
  **Workaround:** Use MPSL_TIMESLOT_HFCLK_CFG_XTAL_GUARANTEED instead of MPSL_TIMESLOT_HFCLK_CFG_NO_GUARANTEE when requesting a timeslot.
```